spec/forms/openid_connect_token_form_spec.rb

```diff
@@ -230,6 +231,25 @@
           end
         end
 
+        context 'with an issued time in the future' do
+          before { jwt_payload[:iat] = Time.zone.now.to_i + 1.minute.to_i }
+
+          it 'is invalid' do
+            expect(valid?).to eq(false)
+            expect(form.errors[:client_assertion]).to include(
+              t('openid_connect.token.errors.invalid_iat'),
+            )
+          end
+        end
+
+        context 'with no issued time' do
+          before { jwt_payload.except!(:iat) }
+
+          it 'is invalid' do
+            expect(valid?).to eq(true)
```

Comment:
this comment threw me... it is the opposite

```suggestion
          it 'is still valid' do
            expect(valid?).to eq(true)
```

Reply:
🤦🏼‍♂️